TMDbLib/Objects/Search/SearchCollection.cs

```diff
@@ -1,26 +1,59 @@
+using System.Collections.Generic;
 using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
 
 namespace TMDbLib.Objects.Search
 {
-    public class SearchCollection
+    public class SearchCollection : SearchBase
     {
+        // Property to hold additional data from the JSON
+        [JsonExtensionData]
+        private IDictionary<string, JToken> _additionalData;
+        
         [JsonProperty("adult")]
         public bool Adult { get; set; }
         
         [JsonProperty("backdrop_path")]
         public string BackdropPath { get; set; }
-
-        [JsonProperty("id")]
-        public int Id { get; set; }
+        
+        private string _name;
+        private string _originalName;
```

Comment:
```suggestion

        private string _name;
        private string _originalName;
```

Why was the `id` field removed?

Reply:
It's inherited from SearchBase